CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+lunch, attend meetups or conferences as a group, communicate on social 
+media, or interact with each other in other communities. 
+
+Abusive or unwelcoming behavior between community members still has a 
+profound impact on individuals and on the community when it happens beyond 
+our walls. The pyOpenSci Executive Director and the Advisory Council will 
+use our discretion when deciding whether to enforce this code of conduct 
+and potentially remove someone from the pyOpenSci community after reports 
+of such behavior happening outside of pyOpenSci, taking into account the 
+impact on the individual community members involved as well as the impact 
+on the community at large.
+
+```{note}
+When in doubt, please report unacceptable behavior to us. If someone's behavior make you feel unsafe within the pyOpenSci space, we want to support and help you. 
+```
+
+## Enforcement
+
+We’ve categorized unacceptable behavior into abuse and unwelcoming behavior in the section above.
+
+### Enforcement related to abusive behavior 
+
+We do not tolerate abusive behavior at pyOpenSci. 
+If we witness or receive a report of abusive behavior, we will 
+contact the perpetrator to have a conversation with them and verify 
+what has transpired. 
+
+We will follow this response protocol:
+
+If we verify abusive behavior:
+
+* They will be removed from the pyOpenSci community
+* If applicable, their employment with pyOpenSci will be terminated. 
+* Their Slack account will be deactivated, and 
+* Permissions will be removed from any pyOpenSci-related repositories and/or accounts. 
+
+The person will not be welcome in any physical or digital spaces covered by the pyOpenSci Code of Conduct.
+
+### Enforcement related to unwelcome behavior 
+If we verify unwelcome, but non-abusive behavior, we will have a 
+conversation with the person about their behavior. The goal of the conversation will be to ensure that they understand the expectation that they not repeat the behavior or other behaviors that would violate the Code of Conduct a second time. 
+
+### Response protocol steps
+The pyOpenSci Executive Director and the Executive Council will use the steps below to respond to reports of code of conduct violations.
+
+1. The report will be read carefully to determine whether there has been a code of conduct violation
+2. We will followup with the person who reported the violation 
+3. We will followup with the community member who violated the code of conduct 
+
+## Reporting code of conduct violations
+
+If you see a violation of our code of conduct, [please report it to our 
+pyOpenSci Code of Conduct Stewards using this form.](https://forms.gle/7MR7tCpDuprtStWP9) This is a short Google Form that will be sent to 
+any Executive Council member acting as a Code of Conduct Steward (see below for details). Alternatively, if you wish to report to one of the Code of Conduct stewards specifically, email them directly.
```

Comment:
Here, it might be worth considering being explicit about who this goes to (e.g. naming folks here). If someone is in a position where they need to submit a CoC report, being very transparent and open about who is responding I think would be appreciated

Reply:
this is a good point. we haven't decided who that is yet @lheagy . i will definitely be on the email (coc@pyopensci.org) but i thought either our executive council @kcranston @tracyteal or some combo of executive and advisory council (which you are on!). I'll bring this up in our board meeting later today.